service/history/transferQueueStandbyProcessor.go

```diff
@@ -293,7 +293,7 @@ func (t *transferQueueStandbyProcessorImpl) processCloseExecution(transferTask *
 		workflowCloseTimestamp := wfCloseTime
 		workflowCloseStatus := getWorkflowExecutionCloseStatus(executionInfo.CloseStatus)
 		workflowHistoryLength := msBuilder.GetNextEventID() - 1
-		workflowExecutionTimestamp := getWorkflowExecutionTimestamp(msBuilder).UnixNano()
+		workflowExecutionTimestamp, visibilityMemo, _ := getWorkflowExecutionTimestampAndMemo(msBuilder)
```

Comment:
Why do we have the third ignored parameter?

Reply:
Used separate functions for clarification.